File: Node/models/transaction.js
class Transaction{
    constructor(from,
                to,
                value,
                fee,
                dateCreated,
                senderPubKey,
                senderSignature,
                transactionHash,
                minedInBlockIndex,
                transferSuccessful){      
                    this.from = from;
                    this.to = to;
                    this.value = value;
                    this.fee = fee;
                    this.dateCreated = dateCreated
                    this.senderPubKey = senderPubKey;
                    this.senderSignature = senderSignature;
                    this.transactionHash = transactionHash;
                    this.minedInBlockIndex = minedInBlockIndex;
                    this.transferSuccessful = transferSuccessful
                }
}

module.exports = Transaction

File: BlockExplorer/BlockExplorer/Models/Transaction.cs
namespace BlockExplorer.Models
{
    using System;

    public class Transaction
    {
        public string From { get; set; }

        public string To { get; set; }

        public double Value { get; set; }

        public int Fee { get; set; }

        public DateTime DateCreated { get; set; }

        public string SenderPubKey { get; set; }

        public string[] SenderSignature { get; set; }

        public string TransactionHash { get; set; }

        public int MinedInBlockIndex { get; set; }

        public bool TransferSuccessful { get; set; }
    }
}
